resources/acquireLock.lua

```diff
@@ -0,0 +1,13 @@
+local lockKey = KEYS[1]
```

Comment:
Not sure what the Lua convention is, but in node it's to have underscore filenames...